README.md

```diff
@@ -314,6 +314,7 @@
 
     // good
     const itemsCopy = [...items];
+    const itemsCopy = items.slice();
```

Comment:
Certainly both of them copy the array - why prefer the older API construct over the newer syntactical one?